_posts/2016-08-09-collections-app.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: post
+title: "Collecting Connections through the Powerbox"
+author: David Renshaw
+authorUrl: https://github.com/dwrensha
+---
+
+Starting work on a project in Sandstorm often means creating many grains, each pertaining to a
+different aspect of your work. In a typical project, you might have a spreadsheet, a chat room, a
+kanban board, and several source code repositories. Sandstorm makes it easy to share these grains
+with your collaborators, but until recently, you would need to share each grain to each collaborator
+separately --- a task that could quickly become tedious. What if you want to share all of the
+project's grains as a single unit?
+
+Now that we have released the
+[Collections app](https://apps.sandstorm.io/app/s3u2xgmqwznz2n3apf30sm3gw1d85y029enw5pymx734cnk5n78h),
+we have a satisfying answer to that question; to share many grains at once, you add them to a collection.
+
+<img src="/news/images/collections-1.png"
+ title="a collection with a chatroom, a kanban board, a spreadsheet, and two git repos">
+
+A collection is a list of grains. Any collaborator with whom you share a collection gets access to
+all of the grains in it. On the flip side, when you remove a grain from the collection, your
+collaborators lose access to that grain. Moreover, when you revoke a collaborator from the
+collection, that (now former-) collaborator loses access to all of the collection's grains.
+
+And since a collection is itself a grain, sharing one works just like sharing anything else on
+Sandstorm, through the "Share access" button.
+
+<img src="/news/images/collections-2.png" title='the "Share access" dialog'>
+
+### Powerbox makes it possible
+
+The fact that we have implemented collections *as an app* may come as a surprise. Indeed, the notion
+of a collection might seem fundamental enough to deserve being baked-in as a core feature of
+Sandstorm. However, Sandstorm bakes in an even more fundamental notion: the idea that grains can
+refer to and coordinate with one another. The embodiment of this idea is the [powerbox](/how-it-works#powerbox),
+an interface
+--- mediated and auditable by the user --- through which grains can exchange capabilities.
```

Comment:
Your hyphens here seem to remain hyphens, and never turned into em dashes, if I understand correctly. It'd be nice to have em dashes. I'm only testing in the GitHub preview; if Jekyll renders them into dashes, then great! Ignore this comment in that cas!